src/main/java/com/javahelps/restservice/CodeReview.java

```diff
@@ -1 +1,42 @@
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.Locale;
 
+public class CodeReview {
+
+    Integer x;
+    String Name;
+    String employee_address;
+
+    // This function is used to pass the employee name
+    public void setName(String newName){
+        if (newName == null || newName == ""){
```

Comment:
you could use instead of equality of null and " " you could use    **newName.Empty()**